portfolio-project-react/src/App.js

```diff
@@ -1,21 +1,18 @@
-import "./index.css";
+import { useState } from "react";
 import Header from "./components/Header/Header";
 import Content from "./components/Content/Content";
 import Footer from "./components/Footer/Footer";
-
-import { useState } from "react";
+import { IsEditModeEnabled } from "./EditModeContext";
```

Comment:
why we have context work in this PR?

Reply:
Using context api for storing a state globally was acceptance criteria of a previous task.

Reply:
Its not a part of this Task then why I am seeing this in this PR? it should be merged into the main branch before creating this PR